snuba/cli/cleanup.py

```diff
@@ -23,7 +23,7 @@ def cleanup(clickhouse_host, clickhouse_port, dry_run, database, dataset, log_le
     from snuba.clickhouse import ClickhousePool
 
     dataset = get_dataset(dataset)
-    table = dataset.get_schema().get_local_table_name()
+    table = dataset.get_write_schema().get_local_table_name()
```

Comment:
Need to figure out what `cleanup.py` does to see if this is the correct schema for this case.